superset/db_engine_specs/hana.py

```diff
@@ -0,0 +1,59 @@
+from superset.db_engine_specs.base import LimitMethod
+from superset.db_engine_specs.postgres import PostgresBaseEngineSpec
+
+
+class HanaPyhdbEngineSpec(PostgresBaseEngineSpec):
+    engine = "hana+pyhdb"
```

Comment:
I believe this should just be `hana` (`pyhdb` in this case is just the driver, i.e. does not need to be specified separately).